ci/build-all-steps.yml

```diff
@@ -84,14 +88,14 @@ steps:
       call install\local_setup.bat
       cd \dev\ros2
       vcs branch
-      ament build ^
-        --parallel ^
+      colcon build ^
+        --merge-install ^
         --cmake-args ^
-          -G "${{ parameters.generator }}" ^
```

Comment:
Looks like colcon-cmake also now supports "Visual Studio 16 2019" so you might as well leave this parameter in for the future

Reply:
I've added this back, but unfortunately if the `VisualStudioVersion` environment variable is not present, `colcon` will fail